File: Entregable4.2/src/main/java/pojo/SalesPerDay.java
package pojo;

import java.util.Date;

public class SalesPerDay {

	private Date date;
	private Long quantity;
	private double amount;
	
	public SalesPerDay() {}
	
	public SalesPerDay(Date date, Long quantity, double amount) {
		this.date = date;
		this.quantity = quantity;
		this.amount = amount;
	}

	public Date getDate() {
		return date;
	}

	public Long getQuantity() {
		return quantity;
	}

	public double getAmount() {
		return amount;
	}
}

File: Entregable5/src/main/resources/static/js/salesPerDay.js
document.addEventListener("DOMContentLoaded", () => {

    const getSalesPerDay = async() => {
        const salesPerDay = document.querySelector("#sales-per-day-list");

        try {
            salesPerDay.innerHTML = `
            <div class="spinner-border text-light" role="status"></div>
            `
            const url = "/tickets/sales-per-day"
            const response = await fetch(url);
            const sales = await response.json();
            salesPerDay.innerHTML = '';

            sales.forEach(({date, quantity, amount}) => {
                salesPerDay.innerHTML += `
                <div class="card text-white bg-dark mb-3">
                    <div class="card-body">
                        <h5 class="card-title">Fecha: ${date}</h5>
                        <h5 class="card-title">Cantidad: ${quantity}</h5>
                        <h5 class="card-title">Monto total: $${amount}</h5>
                    </div>
                </div>
                `   
            })	
        } catch (response) {
            console.log("Algo falló.");
        }
    }

    (async () => {
        try {
            await getSalesPerDay();
        } catch (e) {
        }
    })();
})